pypulseq/make_arbitrary_grad.py

```diff
@@ -45,36 +54,34 @@ def make_arbitrary_grad(
         If slew rate is violated.
         If gradient amplitude is violated.
     """
-    if system == None:
+    if system is None:
```

Comment:
Changing defaults from 0 to None is I think more Pythonic, but also a breaking change if anyone was using these as 0, not that I think many people are using it. Would it make sense to accept both 0 and None in the checks below for some time and remove the check for 0 later?

Reply:
I also think nobody ever sets a value to 0 in order to fall back to the defaults and I don't like it at all, but I agree that it is a breaking change. For the moment, I changed it to `if None or 0` and at some point we can maybe change it in all functions.

I also refactored the slew_rate calculation using `np.diff()`. Cleaner and more pythonic imo. Results are the same.